fdbcli/fdbcli.actor.cpp

```diff
@@ -1493,6 +1497,18 @@ void printStatus(StatusObjectReader statusObj, StatusClient::StatusLevel level,
 				outputString += "\n\nWARNING: A single process is both a transaction log and a storage server.\n  For best performance use dedicated disks for the transaction logs by setting process classes.";
 			}
 
+			std::string ddEnabled;
+			if (statusObjCluster.get("data_distribution", ddEnabled) && ddEnabled == "off") {
+				outputString += "\n\nWARNING: Data distribution is off.";
+			} else {
+				if (statusObjCluster.get("data_distribution_failure_reaction", ddEnabled) && ddEnabled == "off") {
+					outputString += "\n\nWARNING: Data distribution is currently turned on but disabled for all storage server failures.";
+				}
+				if (statusObjCluster.get("data_distribution_rebalancing", ddEnabled) && ddEnabled == "off") {
+					outputString += "\n\nWARNING: Data distribution is currently turned on but shard size balancing is currently disabled.";
+				}
+			}
```

Comment:
Its possible we want all warnings to be shown in the MINIMAL section, but that is out of the scope of this PR

Reply:
I bring it up because this PR is currently changing the behavior by removing the warnings, whereas putting them in minimal would leave the behavior roughly the same.

I'm fine removing them, though, if that's what we want.